packages/ui/src/proposals/components/ProposalList/ProposalListItem.tsx

```diff
@@ -74,18 +82,28 @@ export const ProposalListItem = ({ proposal, isPast, memberId, isCouncilMember }
         </Tooltip>
       </StageField>
       <MemberInfo member={proposal.proposer} memberSize="s" showIdOrText />
-      {isPast && (
-        <StageField>
-          <Subscription>{toDDMMYY(displayDate)}</Subscription>
-        </StageField>
-      )}
+      <StageField>{isPast && <Subscription>{toDDMMYY(displayDate)}</Subscription>}</StageField>
       <StageField>
         <ProposalItemVoteDetails proposal={proposal} memberId={memberId} isCouncilMember={isCouncilMember} />
       </StageField>
+      {hover && (
+        <CopyButton
+          square
+          size="small"
+          textToCopy={getUrl({ route: ProposalsRoutes.preview, params: { id: proposal.id } })}
+          icon={<LinkIcon />}
+        />
+      )}
     </ProposalItem>
   )
 }
 
+const CopyButton = styled(CopyButtonTemplate)`
+  ${css`
+    justify-self: end;
+  `}
```

Comment:
Here there's no need for the `css` tag, you can write css directly in the `styled(CopyButtonTemplate)` tag.

More importantly I would really prefer if the hover logic was handled in css:
```suggestion
  justify-self: end;
  visibility: hidden;
```

And inside the `ProposalItem` declaration add:
```css
&:hover ${CopyButton} {
  visibility: visible;
}
```
(in case there is a delay with hover a `transition: none` on the `CopyButton` would take care of it)
This would really reduce the code complexity for this component.